monai/apps/reconstruction/my_mri_dictionary.py

```diff
@@ -0,0 +1,272 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Dict, Hashable, Mapping, Optional, Sequence
+
+import numpy as np
+from numpy import ndarray
+from torch import Tensor
+
+from monai.apps.reconstruction.my_mri_array import EquispacedKspaceMask, RandomKspaceMask
+from monai.config import DtypeLike, KeysCollection
+from monai.config.type_definitions import NdarrayOrTensor
+from monai.transforms.croppad.array import SpatialCrop
+from monai.transforms.croppad.dictionary import Cropd
+from monai.transforms.intensity.array import NormalizeIntensity
+from monai.transforms.transform import MapTransform
+from monai.utils.type_conversion import convert_to_tensor
+
+
+class RandomKspaceMaskd(MapTransform):
+    """
+    Dictionary-based wrapper of :py:class:`monai.transforms.RandomKspaceMask`.
+
+    Args:
+        keys: keys of the corresponding items to be transformed.
+            See also: monai.transforms.MapTransform
+        center_fractions: Fraction of low-frequency columns to be retained.
+            If multiple values are provided, then one of these numbers is
+            chosen uniformly each time.
+        accelerations: Amount of under-sampling. This should have the
+            same length as center_fractions. If multiple values are provided,
+            then one of these is chosen uniformly each time.
+        spatial_dims: Number of spatial dims (e.g., it's 2 for a 2D data; it's
+            also 2 for psuedo-3D datasets like the fastMRI dataset).
+            The last spatial dim is selected for sampling. For the fastMRI
+            dataset, k-space has the form (...,num_slices,num_coils,H,W)
+            and sampling is done along W. For a general 3D data with the
+            shape (...,num_coils,H,W,D), sampling is done along D.
+        is_complex: if True, then the last dimension will be reserved
+            for real/imaginary parts.
+        seed: Seed for the random number generator. Setting the seed
```

Comment:
seem as the array level transform.

Thanks.

Reply:
Oh if you mean adding `set_random_state`, I just added in my last commit.